pegasus/sites.v3/code.org/public/css/dance-landing/dance-teacher-resources.css

```diff
@@ -6,6 +6,7 @@
   overflow: hidden;
   position: relative;
   float: left;
+  margin-top: 100px;
```

Comment:
You don't need to keep this large margin at the top. It will make a large margin for both of the resource tiles on smaller widths. Instead, I would add a margin-bottom: 20px or 40px to the wide, banner tile.

![image](https://user-images.githubusercontent.com/1920530/48286187-bc580300-e419-11e8-8db7-c428e6ac70e4.png)